cmake/external_mlsl.cmake

```diff
@@ -0,0 +1,76 @@
+# ******************************************************************************
+# Copyright 2017-2018 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ******************************************************************************
+
+# Enable ExternalProject CMake module
+include(ExternalProject)
+
+#------------------------------------------------------------------------------
+# Download MLSL
+#------------------------------------------------------------------------------
+
+SET(MLSL_GIT_URL https://github.com/intel/MLSL)
+SET(MLSL_GIT_TAG master)
```

Comment:
Please select a specific version, not master. If master changes then the build breaks.

Reply:
Done